src/components/Nav.svelte

```diff
@@ -249,18 +272,30 @@
       <div class="col-6 col-sm-6 col-md-9">
         <nav class="nav__links-wrapper">
           <div class="nav__mobile-menu">
-            <input class="nav__mobile-toggle" type="checkbox" />
-            <span class="nav__mobile-toggle-line" />
-            <span class="nav__mobile-toggle-line" />
-            <span class="nav__mobile-toggle-line" />
-            <ul class="nav__links-list">
+            <input class="nav__mobile-toggle" on:click={toggleMobileMenu} />
```

Comment:
If we convert to use JS for nav menu, I think we don't need the `input` anymore? We can wrap the three spans into a `button` and attach the event handler there.